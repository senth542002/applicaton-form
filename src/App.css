.App {
  text-align: center;
}

.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 40vmin;
  pointer-events: none;
}

.App-header {
  background-color: #282c34;
  min-height: 10vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-home-header {
  background-color: #282c34;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-empty-gap {
    min-height: 10vh;
}

.App-results {
  font-family: sans-serif;
  text-align: center;
}

.Application-header {
  min-height: 10vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: left;
  font-size: calc(10px + 2vmin);
  color: black;
}

.App-link {
  color: white;
  align-items: right;
  justify-content: right;
}

.App-title {
  font-size: 1.5em;
}

.App-intro {
  font-size: large;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
