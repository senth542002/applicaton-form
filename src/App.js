import React from 'react'
import './App.css'
import ApplicationForm from './ApplicationForm'

function App () {
  return (
    <div className='App'>
      <header className='App-header'>
        <h2>Application Form</h2>
      </header>
      <div className='Application-header'>
        <ApplicationForm
          style={{
            display: 'flex',
            alignItems: 'left',
            justifyContent: 'left'
          }}
        />
      </div>
      <div className='rows' script='display:inline;' />
    </div>
  )
}

export default App
