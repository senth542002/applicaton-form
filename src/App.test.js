import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import App from './App'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<App />, div)
  ReactDOM.unmountComponentAtNode(div)
})

describe('AppTest', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<App debug />)

    expect(component).toMatchSnapshot()
  })
})

describe('Addition', () => {
  it('knows that 2 and 2  make 4', () => {
    expect(2 + 2).toBe(4)
  })
})
