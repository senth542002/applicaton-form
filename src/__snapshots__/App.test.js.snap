// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTest should render correctly in "debug" mode 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h2>
      Application Form
    </h2>
  </header>
  <div
    className="Application-header"
  >
    <ApplicationForm
      style={
        Object {
          "alignItems": "left",
          "display": "flex",
          "justifyContent": "left",
        }
      }
    />
  </div>
  <div
    className="rows"
    script="display:inline;"
  />
</div>
`;
